samples/javascript_nodejs/14.nlp-with-dispatch/index.js

```diff
@@ -8,46 +8,41 @@ const path = require('path');
 const restify = require('restify');
 
 // Import required bot services. See https://aka.ms/bot-services to learn more about the different parts of a bot.
-const { BotFrameworkAdapter, MemoryStorage, ConversationState, UserState } = require('botbuilder');
-// Import required bot configuration.
-const { BotConfiguration } = require('botframework-config');
+const { BotFrameworkAdapter } = require('botbuilder');
+const { LuisService } = require('botframework-config');
+const { LuisRecognizer, QnAMaker } = require('botbuilder-ai');
+const { DispatchBot } = require('./bots/dispatchBot');
 
-const { DispatchBot } = require('./bot');
-
-// Read botFilePath and botFileSecret from .env file
-// Note: Ensure you have a .env file and include botFilePath and botFileSecret.
+// Note: Ensure you have a .env file and include all necessary credentials to access services like LUIS and QnAMaker.
 const ENV_FILE = path.join(__dirname, '.env');
 require('dotenv').config({ path: ENV_FILE });
 
-// Get the .bot file path
-// See https://aka.ms/about-bot-file to learn more about .bot file its use and bot configuration.
-const BOT_FILE = path.join(__dirname, (process.env.botFilePath || ''));
-let botConfig;
-try {
-    // Read bot configuration from .bot file.
-    botConfig = BotConfiguration.loadSync(BOT_FILE, process.env.botFileSecret);
-} catch (err) {
-    console.error(`\nError reading bot file. Please ensure you have valid botFilePath and botFileSecret set for your environment.`);
-    console.error(`\n - The botFileSecret is available under appsettings for your Azure Bot Service bot.`);
-    console.error(`\n - If you are running this bot locally, consider adding a .env file with botFilePath and botFileSecret.`);
-    console.error(`\n - See https://aka.ms/about-bot-file to learn more about .bot file its use and bot configuration.\n\n`);
-    process.exit();
-}
-
-// For local development configuration as defined in .bot file
-const DEV_ENVIRONMENT = 'development';
-
-// bot name as defined in .bot file or from runtime
-const BOT_CONFIGURATION = (process.env.NODE_ENV || DEV_ENVIRONMENT);
-
-// Get bot endpoint configuration by service name
-const endpointConfig = botConfig.findServiceByNameOrId(BOT_CONFIGURATION);
-
 // Create adapter.
 // See https://aka.ms/about-bot-adapter to learn more about .bot file its use and bot configuration .
 const adapter = new BotFrameworkAdapter({
-    appId: endpointConfig.appId || process.env.microsoftAppID,
-    appPassword: endpointConfig.appPassword || process.env.microsoftAppPassword
+    appId: process.env.microsoftAppID,
+    appPassword: process.env.microsoftAppPassword
+});
+
+const luisService = new LuisService({
+    appId: process.env.DispatchAppId,
+    authoringKey: process.env.DispatchAuthoringKey,
+    region: process.env.DispatchRegion
+});
+
+const dispatchRecognizer = new LuisRecognizer({
+    applicationId: luisService.appId,
+    endpoint: luisService.getEndpoint(),
```

Comment:
Can you instead use the similar code/environment variable from Core-Bot?

Reply:
Ok. I'll change the variable names once the email convo has resolved, since it will determine whether or not the env need to be more specific (i.e. if there are still 3 luis env variables that need to look different from one another)